scripts/perf/dags/elastic_dag.py

```diff
@@ -123,5 +188,11 @@ class DagShape(Enum):
         pass
     elif SHAPE == DagShape.LINEAR:
         chain(*tasks)
+    elif SHAPE == DagShape.BINARY_TREE:
+        chain_as_binary_tree(*tasks)
+    elif SHAPE == DagShape.STAR:
+        chain_as_star(*tasks)
+    elif SHAPE == DagShape.GRID:
+        chain_as_grid(*tasks)
```

Comment:
This list will probably grow over time and pylint will start to moan about to many elif cases. So I would suggest to use dictionary:
```python
shape_function_map = {
    DagShape.LINEAR: chain,
    DagShape.BINARY_TREE: chain_as_binary_tree,
    ...
}
builder = shape_function_map[SHAPE]
builder(*tasks)
```

Reply:
fixed 👍